.docker/scholarly-articles/data/article1/1.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<item xmlns="http://libero.pub">
+
+    <meta>
+
+        <id>article1</id>
+        <service>scholarly-articles</service>
+
+    </meta>
+
+    <front xml:lang="en">
+
+        <title>Article 1</title>
```

Comment:
Should match `.docker/api/data/scholarly-articles/`, but maybe even symlink/mount the same directory.

Reply:
Since we are uncertain about the future of `dummy-api` in this environment, I'll go for duplicating the files.